scripts/plots/nwps/exevs_nwps_wave_grid2obs_plots.sh

```diff
@@ -0,0 +1,213 @@
+	[[ "$LOUD" = YES ]] && set -x
+else
+	set -x
+	echo ' '
+	echo '**************************************** '
+	echo '*** FATAL ERROR: NO NWPS *.stat FILES *** '
+	echo "             for ${VDATE} "
+	echo '**************************************** '
+	echo ' '
+	echo "${MODELNAME}_${RUN} $VDATE $vhour : NWPS *.stat files missing."
+	[[ "$LOUD" = YES ]] && set -x
+	"FATAL ERROR: NO NWPS *.stat files for ${VDATE}"
+	err_exit "FATAL ERROR: Did not copy the NWPS *.stat files for ${VDATE}"
+	exit
+fi
+
+#################################
+## Make the command files for cfp 
+#################################
+
+${USHevs}/${COMPONENT}/evs_wave_timeseries.sh
+
+# lead_averages
+${USHevs}/${COMPONENT}/evs_wave_leadaverages.sh
+
+chmod 775 plot_all_${MODELNAME}_${RUN}_g2o_plots.sh
+
+###########################################
+# Run the command files for the PAST31DAYS 
+###########################################
+
+if [ ${run_mpi} = 'yes' ] ; then
+	mpiexec -np 36 --cpu-bind verbose,core --depth=3 cfp plot_all_${MODELNAME}_${RUN}_g2o_plots.sh
+else
+	echo "not running mpiexec"
+	sh plot_all_${MODELNAME}_${RUN}_g2o_plots.sh
+fi
+
+#####################
+# Gather all the files 
+#######################
+
+periods='LAST31DAYS'
+if [ $gather = yes ] ; then
+	echo "copying all images into one directory"
+	cp ${DATA}/wave/*png ${DATA}/sfcshp/.  ## lead_average plots 
+	nc=$(ls ${DATA}/sfcshp/*.fhrmean_valid*.png | wc -l | awk '{print $1}')
+	echo "copied $nc lead_average plots"
+	for period in ${periods} ; do
+		period_lower=$(echo ${period,,})
+		if [ ${period} = 'LAST31DAYS' ] ; then
+			period_out='last31days'
+		fi
+
+		# check to see if the plots are there
+    	    	nc=$(ls ${DATA}/sfcshp/*${period_lower}*.png | wc -l | awk '{print $1}')
+		echo " Found ${nc} ${DATA}/plots/*${period_lower}*.png files for ${VDATE} "
+		if [ "${nc}" != '0' ]
+		then
+			set -x
+			echo "Found ${nc} ${period_lower} plots for ${VDATE}"
+			[[ "$LOUD" = YES ]] && set -x
+		else
+			set -x
+			echo ' '
+			echo '**************************************** '
+			echo '*** FATAL ERROR: NO ${period} PLOTS  *** '
+			echo "    found for ${VDATE} "
+			echo '**************************************** '
+			echo ' '
+			echo "${MODELNAME}_${RUN} ${VDATE} ${period}: PLOTS are missing."
+			[[ "$LOUD" = YES ]] && set -x
+		
+			"FATAL ERROR : NO ${period} PLOTS FOR ${VDATE}"
+			err_exit "FATAL ERROR: Did not find any ${period} plots for ${VDATE}"
+		fi    	    	                                                                                                                                                                                                    		
+		# tar and copy them to the final destination
+
+		if [ "${nc}" > '0' ] ; then
+			cd ${DATA}/sfcshp
+			tar -cvf evs.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.${period_out}.v${VDATE}.tar evs.*${period_lower}*.png
+		fi
+	done
+	cpreq evs.${STEP}.${COMPONENT}.${RUN}.*.tar ${COMOUTplots}/.
```

Comment:
1. Replace `cpreq` with `cp -v`
2. Add check for `evs.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}.${period_out}.v${VDATE}.tar` and copy that tar file if it exists